packages/cli/src/cli/overview/action.create.overview.ts

```diff
@@ -0,0 +1,176 @@
+import { LogConfig, LogType } from '@basemaps/shared';
+import * as path from 'path';
+import { promises as fs } from 'fs';
+import { CommandLineAction, CommandLineIntegerParameter, CommandLineStringParameter } from '@rushstack/ts-command-line';
+import { Bounds, NamedBounds, QuadKey, TileMatrixSet, TileMatrixSets } from '@basemaps/geo';
+import os from 'os';
+import tar from 'tar';
+import { WorkerRpcPool } from '@wtrpc/core';
+import { JobTiles } from './tile.generator.js';
+import { CotarIndexBinary, CotarIndexBuilder, CotarIndexOptions, TarReader } from '@cotar/core';
+import { SourceMemory, ChunkSource } from '@chunkd/core';
+import { fsa } from '@chunkd/fs';
+import { CogBuilder } from '../../cog/builder.js';
+import { filterTiff, MaxConcurrencyDefault } from '../../cog/job.factory.js';
+import { Cutline } from '../../cog/cutline.js';
+import { CogTiff } from '@cogeotiff/core';
+import { createHash } from 'crypto';
+
+// Create tiles per worker invocation
+const WorkerTaskSize = 500;
+const workerUrl = new URL('./tile.generator.js', import.meta.url);
+const threadCount = os.cpus().length / 8;
+const pool = new WorkerRpcPool(threadCount, workerUrl);
```

Comment:
Can you add typing to this

```typescript
export type RpcContract = {
   createTiles(x: something): something;
}

const pool = WorkerRpcPool<RpcContract>
```

Reply:
I am not sure I am follow this? Is the `RpcContract` same as the `WorkerRpc` in the `tile.generator.ts`? So we can directly pass it into pool and remove the `workerUrl` things?